### PR TITLE
added Cl and S photochemistry

### PR DESCRIPTION
**Pull request by simoncorrera**
I have added in the species HCl, DCl, Cl, ClO, ClCO, S, SO, SO2, SO3, H2SO4, and HDSO4 as part of my project for the Boulder Solar alliance REU. I have added new neutral reactions for these species, Added in photolysis reactions for HCl, DCl, SO2, SO3, H2SO4, and HDSO4, and added density boundary conditions for HCl, and SO2. I have attached the reaction network spreadsheet as well for clarity since git hub has trouble showing those changes. All of the HDSO4 reactions used mass scaling. Some of the Cl and D  reactions used mass scaling, but some were input incorrectly in the excel file where the power was not included. Some of these did  have real data, but mass scaling was used instead because the real data seemed to give unexpected results.
[REACTION_NETWORK_VENUS.xlsx](https://github.com/user-attachments/files/16474745/REACTION_NETWORK_VENUS.xlsx)

### CODE REVIEW

File: MODEL_SETUP.jl [hunk 1]
```diff
@@ -375,6 +377,12 @@ elseif planet=="Venus"
                                     :O2=>Dict("n"=>[3e-3*ntot_at_lowerbdy, NaN], "f"=>[NaN, 0.]),
                                     :N2=>Dict("n"=>[0.032*ntot_at_lowerbdy, NaN]),
 
+                                    #Krasnopolsky, 2010a: this was 400ppb at 74km in altitude, and the actual number is likely lower (is either 4.0E-7, or 4.8E-7 depending on the calculation); and according to Zhang 2012 it is 3.66e-7
+                                    :HCl=>Dict("n"=>[3.66e-7 * ntot_at_lowerbdy, NaN]),
```
Comment (emcangi) [inline MODEL_SETUP.jl:381]: This looks good, it also agrees with https://www.sciencedirect.com/science/article/pii/S0032063314004061?via%3Dihub which is a more recent paper and probably the right one to use (see Figure 5, I'm looking in between the 0-30 and 30-70 lat lines)

File: MODEL_SETUP.jl [hunk 2]
```diff
@@ -375,6 +377,12 @@ elseif planet=="Venus"
...
+
+                                    #Denis A. Belyaev 2012: this was 0.1 ppmv at 165–170 K to 0.5–1 ppmv at 190–192 K; It said 0.1ppm was related to the most common temperature reading so I went with that (this is either 1E-7 or 6.79E-8 depending on the calculation)
+                                    :SO2=>Dict("n"=>[1.0E-7 * ntot_at_lowerbdy, NaN]),
```
Comment (emcangi) [inline MODEL_SETUP.jl:384]: Is this one using the model fit they did with just SO2 or with SO2 and SO together? I noticed those are pretty different

**Comment by simoncorrera** [comment]
I got those numbers for SO2 from section 4.3 of that paper, and it was analyzing figure 8. In figure 8 it says, "values above 85 km are from SPICAV UV". TBH I think the number I put in might be lower than it should be since I just assumed since the coldest temperature had the most data it was the closest to the real value, but that's probably not completely accurate
________________________________

**Review by emcangi — APPROVED** [review]
Looks good, there are some minor things to clean up in the reaction network spreadsheet but I'll handle it after merging

**Pull request merged**